src/client/content_cao.cpp

```diff
@@ -1832,10 +1866,42 @@ void GenericCAO::processMessage(const std::string &data)
 		updateAnimationSpeed();
 	} else if (cmd == AO_CMD_SET_BONE_POSITION) {
 		std::string bone = deSerializeString16(is);
-		v3f position = readV3F32(is);
-		v3f rotation = readV3F32(is);
-		m_bone_position[bone] = core::vector2d<v3f>(position, rotation);
-
+		BoneOverride *previous = m_bone_override[bone];
+		BoneOverride *override = new BoneOverride();
+		override->position.vector = readV3F32(is);
+		override->rotation.next = core::quaternion(readV3F32(is) * core::DEGTORAD);
+		if (is.eof()) {
+			override->position.absolute = true;
```

Comment:
Please include a note for the current protocol version. Even though this does not need a bump, a rough timestamp for this legacy code would be helpful.

Also needs an entry in `networkprotocol.h`.